### DISCUSSION TITLE
Can I multiply size of any image

### DISCUSSION THREAD

**Discussion by GreDiSe** [igordanchenko/react-photo-album#106] [Q&A] [answered]
Hello,

Is it possible to increase size of first image, like in AirBnb?
<img width="1169" alt="Screenshot 2023-03-06 at 15 38 52" src="https://user-images.githubusercontent.com/29945710/223126821-08eff402-fe1f-48f3-8b16-f279aa263d72.png">

**Comment by igordanchenko** [comment, marked as answer]
Unfortunately, not. This library doesn't support this kind of layout algorithm.
